target/linux/lantiq/patches-5.10/0019-MTD-nand-remove-hardcoded-soft-ecc.patch

```diff
@@ -0,0 +1,11 @@
+--- a/drivers/mtd/nand/raw/xway_nand.c
++++ b/drivers/mtd/nand/raw/xway_nand.c
```

Comment:
I think these patches should have a header to explain what they do.

Reply:
I agree with @adschm  it would be nice to have a title why this patch is needed.